### PR TITLE
Samples: Add Blazor sample

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Adding a sample on how to use CosmosClient with a Blazor application

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by ealsur** [comment]
Similar to the closed https://github.com/Azure/azure-cosmos-dotnet-v3/pull/981

The issue is, this does not cover the scenario (that does not work) for which Blazor is different from a normal ASP.NET Core application (WebAssembly).

File: Microsoft.Azure.Cosmos.Samples/Usage/Blazor/BlazorCosmosApp.csproj [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.8.0" />
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/Blazor/BlazorCosmosApp.csproj:8]: Can the version be "*"

File: Microsoft.Azure.Cosmos.Samples/Usage/Blazor/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿#Blazor sample
+This contains a sample of a Blazor application using Cosmos DB.
+
+The application reads all the items from a container, and allows the user to create new items in the container with a button click.
+
+## Setup
+
+1. Update the "Microsoft.Azure.Cosmos.Samples\Usage\Blazor\appsetting.json" with the correct connection string.
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/Blazor/ReadMe.md:8]: Do credentials leak to CX? 

Better example might be to dynamically get a token (SAS, AAD, or ResourceToken) and then bootstrap clent?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos.Samples/Usage/Blazor/ReadMe.md:8]: End user AAD integration is best for these scenarios.

**Pull request closed without merging**